src/seedu/addressbook/AddressBook.java

```diff
@@ -454,15 +455,31 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
      */
```

Comment:
did you update header comment and the Usage variables (help) to say that find is now case insensitive?